src/styles/blocks/stars.css

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+  margin-right: 4px;
+  /* justify-content: space-between; */
+
+  min-width: 96px;
```

Comment:
you don't need this property, since you've added margins

Reply:
)))
done